examples/separate-config/every-feature-multiple-dest-multiple-ms-fixme.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
```

Comment:
This example is intended to be multiple instances of Multiselect, each with multiple destinations. This has not been finished yet, as you can probably see.